File: rust/tcl-registry/src/commands/iapps/tmsh__pwd.rs
//! `tmsh::pwd` command.
use crate::prelude::*;
const FORMS: &[FormSpec] = &[FormSpec {
    kind: FormKind::Default,
    synopsis: "tmsh::pwd",
}];

pub fn spec() -> CommandSpec {
    CommandSpec {
        name: "tmsh::pwd",
        dialects: Some(DialectSet::IAPPS),
        arity: Arity::exact(0),
        hover: Some(HoverSnippet::brief(
            "Returns the current working directory.",
            &["tmsh::pwd"],
            "F5",
        )),
        forms: FORMS,
        ..CommandSpec::DEFAULT
    }
}

File: core/commands/registry/iapps/tmsh__pwd.py
"""tmsh::pwd -- F5 TMSH directory command."""

from __future__ import annotations

from .._base import CommandDef
from ..models import CommandSpec
from ._base import _tmsh_spec, register


@register
class TmshPwd(CommandDef):
    name = "tmsh::pwd"

    @classmethod
    def spec(cls) -> CommandSpec:
        return _tmsh_spec(
            "tmsh::pwd",
            "Returns the current working directory.",
            "tmsh::pwd",
            max_args=0,
        )
